site/en/docs/extensions/mv3/tut_websockets/index.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Using WebSockets in Service Workers'
+seoTitle: 'How to use WebSockets in Chrome Extension Service Workers'
+date: 2023-07-04
+description: 'Step-by-step instructions on how to connect to a WebSocket in your Chrome Extension.'
+---
+
+This tutorial demonstrates how to connect to a WebSocket in your Chrome Extension's service worker. You can find a [working example on Github](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/tutorial.websockets).
+
+## Background
+
+Starting with Chrome M116, extension service workers get full support for WebSockets. Previously, a service worker could become inactive if the WebSocket connection was active for more than 30s and no other extension events occurred at that time, which could result in connection loss and the inability to receive messages.
```

Comment:
Similar to Patrick's comment below, I wouldn't say that the WebSocket being active was a prerequisite to the termination. Maybe something like this would be better:

```suggestion
Starting with Chrome 116, extension service workers get improved support for WebSockets. Previously, a service worker could become inactive despite a WebSocket connection being active if no other extension events occurred for 30 seconds, which would result in the service worker being terminated and the WebSocket connection being closed.
```

Reply:
Thanks, used your suggestion!